pkg/chunk/gcp/bigtable_object_client.go

```diff
@@ -65,6 +68,10 @@ func (s *bigtableObjectClient) PutChunks(ctx context.Context, chunks []chunk.Chu
 
 	for tableName := range keys {
 		table := s.client.Open(tableName)
+		if s.writeLimiter.Allow() == false {
+			s.writeLimiter.WaitN(ctx, len(muts))
```

Comment:
`len(muts)` doesn't look correct to me. Shouldn't be `len(muts[tableName])` to rate limit on the mutations? (assuming we want to limit on the mutations).

Reply:
Good catch, it should be len(muts[tableName]) to rate limit on the mutations.